tests/simple_der.rs

```diff
@@ -0,0 +1,39 @@
+pub use asn1rs::prelude::*;
+
+asn_to_rust!(
+    r"Data-Structures DEFINITIONS ::=
+BEGIN
+  DataStructures ::= SEQUENCE
+  {
+     int              INTEGER,
+     limitedString    UTF8String (SIZE(1..16)),
+     optionalString   UTF8String OPTIONAL,
+     enumerated       ENUMERATED {value1, value2, value3},
+     optionalChoice   CHOICE
+     {
+        int1   [0] INTEGER,
+        int2   [1] INTEGER
+     }  OPTIONAL,
+     sequenceOfString SEQUENCE OF UTF8String
+  }
+END"
+);
+
+#[test]
+fn simple_der() {
+    let der_content = b"0'\x02\x0209\x0c\tSomething\n\x01\x00\xa1\x05\x02\x03\x00\xd410\x0c\x0c\x04abcd\x0c\x04efgh".to_vec();
```

Comment:
Thanks for changing the example. In the long term, I prefer this to have the same notation as in the [remaining crate](https://github.com/kellerkindt/asn1rs/blob/master/tests/basic_ia5string.rs#L36)